Day8-Assignment/src/Components/CardContainer.js

```diff
@@ -0,0 +1,14 @@
+import NoResultComponent from './NoResultComponent';
+import CardComponent from './CardComponent';
+
+const CardContainer = ({ filteredUser }) => {
+  if (!filteredUser.length) return <NoResultComponent />
+  else {
```

Comment:
no need of else here. It will check the if condition if it doesn't satisfy it will automatically go for return